src/components/SpeciesTable.js

```diff
@@ -386,12 +391,20 @@ const EditModal = ({
   };
 
   const handleEditDetailClose = () => {
+    setError(undefined);
     setIsEdit(false);
     setSpeciesEdit(undefined);
   };
 
   const handleSave = async () => {
-    if (nameSpecies.includes(speciesEdit.name.toLowerCase().trim())) {
+    let editName = speciesEdit.name.toLowerCase().trim();
```

Comment:
We should use `const` instead of `let` if the variable doesn't change after assignment.